core/src/main/java/com/maxshop/util/Format.kt

```diff
@@ -0,0 +1,3 @@
+package com.maxshop.util
+
+fun String.Companion.formatPrice(price: Number) = String.format("%.2f", price)
```

Comment:
Redundant method and file. Remove it please

Reply:
fixed